core/domain/classifier_services.py

```diff
@@ -356,3 +356,75 @@ def delete_classifier_training_job(job_id):
         classifier_models.ClassifierTrainingJobModel.get(job_id))
     if classifier_training_job_model is not None:
         classifier_training_job_model.delete()
+
+
+def get_classifier_exploration_mapping(
+        exp_id, exp_version, state_name, classifier_id):
+    """Gets a classifier exploration mapping domain object from attributes.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. The exploration version.
+        state_name: str. The state to which the classifier belongs.
+        classifier_id: str. ID of the classifier.
+
+    Returns:
+        ClassifierExplorationMapping. Domain object for the classifier
+            exploration mapping.
+    """
+    return classifier_domain.ClassifierExplorationMapping(
+        exp_id, exp_version, state_name, classifier_id)
+
+
+def get_classifier_from_exploration_attributes(exp_id, exp_version,
+                                               state_name):
+    """Gets the classifier object from the exploration attributes.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. The exploration version.
+        state_name: str. The name of the state to which the classifier
+            belongs.
+
+    Returns:
+        ClassifierData. Domain object for the Classifier model.
+
+    Raises:
+        Exception: Entity for class ClassifierExplorationMapping with id not
+            found.
+        Exception: Entity for class ClassifierData with id not found.
+    """
+    classifier_exploration_mapping_model = (
+        classifier_models.ClassifierExplorationMappingModel.get_model(
+            exp_id, exp_version, state_name))
+    classifier_id = classifier_exploration_mapping_model.classifier_id
+    classifier = get_classifier_by_id(classifier_id)
+    return classifier
+
+
+def create_classifier_exploration_mapping(exp_id, exp_version, state_name,
+                                          classifier_id):
+    """Creates an entry for Classifier Exploration Mapping in datastore.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. The exploration version.
+        state_name: str. The state to which the classifier belongs.
+        classifier_id: str. ID of the classifier.
+
+    Returns:
+        mapping_id : str. ID of the classifier exploration mapping instance.
+    """
+    classifier_exploration_mapping_model = (
+        classifier_models.ClassifierExplorationMappingModel.get_model(
+            exp_id, exp_version, state_name))
+    get_classifier_by_id(classifier_id)
+
+    if classifier_exploration_mapping_model is None:
```

Comment:
@seanlip Right now, this if clause just verifies if the model doesn't exists (since we are creating a model). If some other model already exists, we can't create another one. Rather, the else condition should be used when we update the mapping (probably in the future, not for now). The 'if' clause is not necessary here, but still verifies if the mapping model is new and does not exist. Should I remove it?